python/ray/autoscaler/v2/scheduler.py

```diff
@@ -209,7 +246,74 @@ def _compute_score(self) -> UtilizationScore:
         Returns:
             A utilization score for this node.
         """
-        pass
+
+        # Compute the number of resource types being scheduled.
+        num_matching_resource_types = 0
+        for req in self.sched_requests:
+            for resource_name in req.resources_bundle.keys():
+                if resource_name in self.total_resources:
```

Comment:
I think it will never be false?

Reply:
why so? could there be a resource bundle e.g. ('some-unknown-resource') that's not present in the total resources of a node?